### ISSUE TITLE
Crash with ” character

### ISSUE THREAD

**Issue by edreamleo** [jacktasia/fstringify#9]
fstringify 0.1.6, Windows 10.

A file (leo/commands/editCommands.py) containing the ” character causes a crash. The file starts with "# -*- coding: utf-8 -*-" and contains these lines:
```python
if open_curly:
    ch = '‘' if ch == "'" else "“"
else:
    ch = '’' if ch == "'" else "”"
```
The last line causes the crash. Here is the traceback:
```
Traceback (most recent call last):
  File "c:\users\edreamleo\anaconda3\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\users\edreamleo\anaconda3\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\edreamleo\Anaconda3\Scripts\fstringify.exe\__main__.py", line 9, in <module>
  File "c:\users\edreamleo\anaconda3\lib\site-packages\fstringify\__init__.py", line 31, in main
    fstringify(args.src, verbose=args.verbose, quiet=args.quiet)
  File "c:\users\edreamleo\anaconda3\lib\site-packages\fstringify\api.py", line 65, in fstringify
    fstringify_files(files, verbose=verbose, quiet=quiet)
  File "c:\users\edreamleo\anaconda3\lib\site-packages\fstringify\api.py", line 38, in fstringify_files
    changed = fstringify_file(file_path)
  File "c:\users\edreamleo\anaconda3\lib\site-packages\fstringify\api.py", line 15, in fstringify_file
    contents = f.read()
  File "c:\users\edreamleo\anaconda3\lib\encodings\cp1252.py", line 23, in decode
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
UnicodeDecodeError: 'charmap' codec can't decode byte 0x9d in position 67312: character maps to <undefined>
```

**Comment by jacktasia** [comment]
Thanks for opening this! It might take me a while to get to this, but I've added the "help wanted" label if anyone would like to open a PR.

**Comment by jacktasia** [comment]
I think I fixed this in version 0.1.14. Please re-open/comment if you still have issues. Thanks!
